.github/workflows/publish-core-images.yaml

```diff
@@ -1,8 +1,12 @@
 name: Publish Katib Core Images
 
 on:
-  - push
-  - pull_request
+  push:
+    paths-ignore:
+      - 'pkg/new-ui/v1beta1/frontend/**'
+  pull_request:
+    paths-ignore:
+      - 'pkg/new-ui/v1beta1/frontend/**'
```

Comment:
We need to check whether building the UI image supports multi-platform.
it might be better to separate the job to build the UI image.

This means creating the following new actions file.

```yaml
name: Publish Katib UI Image

on:
  push:
  pull_request:

jobs:
  ui:
    name: Publish Image
    uses: ./.github/workflows/build-and-publish-images.yaml
    with:
      component-name: katib-ui
      platforms: linux/amd64,linux/arm64
      dockerfile: cmd/new-ui/v1beta1/Dockerfile
    secrets:
      DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
      DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
```

Reply:
Just to be sure, we should keep the `paths-ignore` changes made to `publish-core-images.yaml`, right?

Reply:
Also, a small correction on the new workflow file
```
on:
  - push
  - pull_request
```

Reply:
> Just to be sure, we should keep the paths-ignore changes made to publish-core-images.yaml, right?

Yes, that's right.

> Also, I 'll add a small correction on the new workflow file

I'm ok.

Reply:
I just pushed the branch with the changes included.